graph-kiota/httpCore.md

```diff
@@ -0,0 +1,33 @@
+### IHttpCore
```

Comment:
this is all already documented [here](https://github.com/microsoft/kiota/blob/main/docs/kiotaabstractions.md#http-core) and I'd much rather have that information at this place so it's easier to find for people outside of Microsoft. Don't hesitate to go PR it.